org-formation/650-identity-providers/microsoft-defender-cloud-roles.yaml

```diff
@@ -0,0 +1,1436 @@
+      offering
+    AllowedPattern: '[-_a-zA-Z0-9]+'
+    Default: DefenderForCloud-DefenderForServers-stackArmor
+  DefenderForServersVmScannerRoleName:
+    Type: String
+    Description: Provide a role ARN name DefenderForCloud-AgentlessScanner for Servers
+      agentless scanning offering
+    AllowedPattern: '[-_a-zA-Z0-9]+'
+    Default: DefenderForCloud-AgentlessScanner-stackArmor
+  ArcAutoProvisioningRoleName:
+    Type: String
+    Description: Provide a role ARN name DefenderForCloud-ArcAutoProvisioning for
+      needed offerings
+    AllowedPattern: '[-_a-zA-Z0-9]+'
+    Default: DefenderForCloud-ArcAutoProvisioning-stackArmor
+  SetupType:
+    Type: String
+    AllowedValues:
+      - Local
+      - Organizational
+    Default: Organizational
+    Description: '(Required) Specifies the type of the Setup: either local or organizational.'
+  ConfigurationID:
+    Type: String
+    Default: MDCSetup
+    Description: (Required) Unique identifier of the deployed configuration.
+  IsPolicyAttachAllowed:
+    Type: String
+    Default: 'true'
+    AllowedValues:
+      - 'true'
+      - 'false'
+    Description: (Optional) Whether MDc setup is allowed to attach policies to existing
+      Instance profiles
+  ProvidedInstanceProfileName:
+    Type: String
+    Default: ''
+    Description: (Optional) Specifies the instance profile Name provided by the user
+      when SetupType=Local.
+  ProvidedAssumeRoleArn:
+    Type: String
+    Default: '*'
+    Description: (Optional) Specifies the automation assume role Arn provided by the
+      user when SetupType=Local.
+  TargetType:
+    Type: String
+    Default: '*'
+    AllowedValues:
+      - Tags
+      - InstanceIds
+      - '*'
+      - ResourceGroups
+    Description: (Optional) Specifies the way in which instances are targeted - applies
+      only for local MDCSetup.
+  TargetInstances:
+    Type: String
+    Default: '*'
+    Description: (Optional) Specifies the instances to be targeted when SetupType=Local
+      and TargetType=InstanceIds.
+  TargetTagKey:
+    Type: String
+    Default: ''
+    Description: (Optional) Specifies the tag key of instances to be targeted when
+      SetupType=Local and TargetType=Tags
+  TargetTagValue:
+    Type: String
+    Default: ''
+    Description: (Optional) Specifies the tag value of instances to be targeted when
+      SetupType=Local and TargetType=Tags
+  ResourceGroupName:
+    Type: String
+    Default: ''
+    Description: (Optional) Specifies the resource group name to be targeted when
+      SetupType=Local and TargetType=ResourceGroups
+  DataSecurityPostureDbRoleName:
+    Type: String
+    Description: Provide a role ARN name DefenderForCloud-DataSecurityPostureDB for
+      Databases DSPM offering
+    AllowedPattern: '[-_a-zA-Z0-9]+'
+    Default: DefenderForCloud-DataSecurityPostureDB-stackArmor
+  IsManagementAccountOnboarding:
+    Type: String
+    Default: 'true'
```

Comment:
@frsfx should this be `false` since we are deploying this template to an AWS member account in our organization?

Reply:
Yes this should be false.

Reply:
Can I give you a new template?

Reply:
done

Reply:
> Can I give you a new template?

why @frsfx?  is there something wrong with the one in this PR?